docs/source/finetune.md

```diff
@@ -0,0 +1,40 @@
+# Finetune Infra
+
+This repository provides the infrastructure for finetuning models using different hardware accelerators such as QAIC, CUDA, and CPU.
```

Comment:
Remove references to CUDA.
We can mention that the same CLI can be used to run FT on GPU by setting the device as ...